implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -523,6 +513,13 @@ impl NodeState {
         self.path.join("policies_storage.lmdb")
     }
 
+    pub fn authorities(&self) -> Result<AuthoritiesConfig> {
```

Comment:
this is ok now.
In a next iteration we can re visit how this authority file is used and for what,  given than the same info is present on the project' information  (project.json right now).